tests/ert/ui_tests/cli/test_field_parameter.py

```diff
@@ -33,13 +33,13 @@ def test_field_param_update_using_heat_equation(heat_equation_storage):
         prior_covariance = np.cov(
             prior_result.values.reshape(
                 prior.ensemble_size, param_config.nx * param_config.ny * param_config.nz
-            )
+            ).T
```

Comment:
Apparently, you can specify `np.cov(A, rowvar=False)` and thus avoid transposing.

Reply:
That is better, thanks. I've pushed a fix.